.github/sync-repo-settings.yaml

```diff
@@ -37,6 +37,7 @@ branchProtectionRules:
     - "units (11)"
     - "Kokoro - Test: Integration"
     - "cla/google"
+    - "Kokoro - Test: Samples"
```

Comment:
nit: move it together with the other Kokoro?

Reply:
Sg, done.